lib/fog/kubevirt/compute/compute.rb

```diff
@@ -371,40 +356,81 @@ def create_client_from_token(url, version)
               :bearer_token => @kubevirt_token
             }
           }
+          version = detect(url.to_s, @opts[:ssl_options])
```

Comment:
lines 359-364 repeat themselves in the `create_client_from_config` and till the end of the method, also the code looks the same (except the kubeclient-client instantiation) - can it be extracted to a method ?

Reply:
Will check